rules/apps.libsonnet

```diff
@@ -66,7 +66,7 @@
               sum by (namespace, label_name) (
                   sum(kube_pod_container_resource_requests_memory_bytes{%(kubeStateMetricsSelector)s} * on (endpoint, instance, job, namespace, pod, service) group_left(phase) (kube_pod_status_phase{phase=~"Pending|Running"} == 1)) by (namespace, pod)
                 * on (namespace, pod)
-                  group_left(label_name) kube_pod_labels{%(kubeStateMetricsSelector)s}
+                  group_left(label_name) max by(namespace, pod, label_name) (kube_pod_labels{%(kubeStateMetricsSelector)s})
```

Comment:
I think this is a leftover, but the `label_name` label is a practice that leaked in here from Grafana's internal practices. We've removed this in other places, so we can get rid of this join all together and just perform `max by(namespace, pod, container) (...)` on the `kube_pod_container_resource_requests_memory_bytes` and `kube_pod_status_phase` metrics.

Reply:
:+1: